task/runner.go

```diff
@@ -475,9 +475,19 @@ func errorInfo(err error) *data.ErrorInfo {
 func humanizeCatalystError(err error) error {
 	errMsg := strings.ToLower(err.Error())
 
+	fileNotAccessibleErrs := []string{
+		"504 Gateway Timeout",
+		"giving up after",
+	}
 	// General errors
-	if strings.Contains(errMsg, "import request") && strings.Contains(errMsg, "504 Gateway Timeout") {
-		return errors.New("file could not be imported from URL because it was not accessible")
+	// TODO(yondonfu): This string matching is ugly and we should come up with a better less error-prone way to
```

Comment:
Agreed. We should probably work with error codes like mediaconvert does (but may always have some string parsing fallback anyway, just hopefully won't be the main logic)